dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/DefaultTrackedEntityService.java

```diff
@@ -546,7 +552,7 @@ private void mapRelationshipItems(
   }
 
   private void mapRelationshipItems(TrackedEntity trackedEntity, boolean includeDeleted)
-      throws ForbiddenException, NotFoundException {
+      throws NotFoundException {
```

Comment:
Was the forbidden exception just not thrown?

Reply:
yes, that was a linting error that I fixed.